docusaurus-search-local/src/client/theme/SearchBar/SearchBar.tsx

```diff
@@ -280,7 +281,9 @@ export default function SearchBar({
               const a = searchFooterLinkElement({ query, isEmpty });
               const div = document.createElement("div");
               div.className = styles.hitFooter;
-              div.appendChild(a);
+              if(removeSeeAllResults === false){
+                div.appendChild(a);
```

Comment:
For me, the main issue currently is when I enter some words in the search box, then press the tab key, it jumped to one of the search target page instead of switching between result items. This won't fix it, will it?

Reply:
Yeah, this won't fix the tab issue. That functionality is in the @easyops-cn/autocomplete.js library, if I'm not mistaken.

Reply:
Then I'm not sure what this PR fixes